### PR TITLE
Fix netplan configuration

### PR DESCRIPTION
**Pull request by vishesh92**
* `gateway4` has been deprecated. Update netplan config to use default routes.
* enable `dhcp4` to fix connectivity to internet

### CODE REVIEW

File: 1-user.md [hunk 1]
```diff
@@ -86,11 +86,13 @@ any yaml file at /etc/netplan: (change interface names, ip ranges accordingly)
...
            addresses: [192.168.122.10/24]
-           gateway4: 192.168.122.1
+           routes:
+             - to: default
+               from: 192.168.122.1
            nameservers:
              addresses: [1.1.1.1,8.8.8.8]
            interfaces: [ens3]
-           dhcp4: false
+           dhcp4: true
```
Comment (rohityadavcloud) [inline 1-user.md:95]: We don't need this if we want to setup static IP, `192.168.122.10/24` in the example.

Comment (vishesh92) [reply 1-user.md:95]: I am not able to access internet from inside the VM. Let me debug the exact issue and fix this.

Comment (rohityadavcloud) [reply 1-user.md:95]: @vishesh92 it's possible you'll need to check/fix the static IP and gateway. Depending on whether this is a VM on KVM, VirtualBox, VMware workstation/fusion etc. some of those require the nic to be activated (which means sometimes restarting the VM or doing a ping to the gateway).

Comment (rohityadavcloud) [inline 1-user.md:95]:
```suggestion
           dhcp4: false
```

**Pull request merged**